ansible/roles/airflow/files/airflow-dags/canning.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
I wonder if actually all the files listed in `airflow-dags` should actually not just live inside of ooni-pipeline. If not what is the reason for doing so?